src/providers/HotspotProvider.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { createContext, ReactNode, useContext } from 'react'
+import { State } from 'react-native-ble-plx'
+import useHotspot from '../utils/useHotspot'
+
+const initialState = {
+  getState: async () => State.Unknown,
+  enable: async () => {},
+  scanForHotspots: async () => {},
+  connectAndConfigHotspot: async () => {},
+  hotspotDetails: undefined,
+  connectedHotspot: undefined,
+  availableHotspots: {},
+}
+
+const HotspotContext = createContext<ReturnType<typeof useHotspot>>(
+  initialState,
+)
+const { Provider } = HotspotContext
+
+const HotspotProvider = ({ children }: { children: ReactNode }) => {
```

Comment:
it might be a little unclear just from a naming perspective that this is providing a context for the hotspot that you're currently connected to over bluetooth

Reply:
I can definitely see that. Any suggestions? `HotspotBleProvider`, `HotspotDeviceProvider`, ? @allenan

Reply:
maybe `ConnectedHotspotProvider`?

Reply:
Cool. I'll go with that.